### PR TITLE
Makes pickaxe mining less miserable

### PR DESCRIPTION
**Pull request by RDS88-beeman**
## About The Pull Request
This makes all pickaxes (not drills, not PKAs, not crushers, not resonators, just pickaxes) mine faster, at the cost of requiring stamina to use.
Standard pickaxes deal five damage per picking action, clamped to 75, while mining as fast as silver pickaxes.
Silver pickaxes are the same, but deal three damage per action.
Diamond pickaxes also deal three damage per action, but are "fast as fuck".
Ignore the glitter, that's being atomised.

## Why It's Good For The Game
Ruko told me to and I trust his game design judgement. I also play ashwalker, and pickaxe mining as an ashwalker is miserable. This helps that take less time, while also making it not a unilateral buff.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

I personally have gotten away with not providing testing evidence for a lot of PR's by going "just trust me."
However, I still don't have any way of recording testing video or taking images, so you are just going to have to trust me on this one. I solemnly swear I did in fact take this into a private server and make sure it all worked.

</details>

## Changelog
:cl:
balance: Pickaxes are faster now, you can put your back into it! Granted, putting your back into it tends to tire you out, so watch out for that.
/:cl:

### CODE REVIEW

**Comment by MarkusLarsson421** [comment]
Get some testing evidence. smh

**Comment by Rukofamicom** [comment]
This does make normal pickaxes (0.5) technically faster than normal drills (0.6), but drills don't have the stamina cost.

**Comment by Rukofamicom** [comment]
dreamseeker_guLi7KO0jE

dreamseeker_DqPCZs4ecG

**Comment by Rukofamicom** [comment]
I do really recommend sharex for testing evidence. It's what I just used to produce those.

File: code/modules/mining/equipment/mining_tools.dm [hunk 1]
```diff
@@ -14,11 +14,21 @@
 	w_class = WEIGHT_CLASS_BULKY
 	custom_materials = list(/datum/material/iron=2000) //one sheet, but where can you make them?
 	tool_behaviour = TOOL_MINING
-	toolspeed = 1
+	toolspeed = 0.5
+	var/stamina_use = 6
```
Comment (Rukofamicom) [inline code/modules/mining/equipment/mining_tools.dm]:
```suggestion
	var/stamina_use = 5
```

File: code/modules/mining/equipment/mining_tools.dm [hunk 2]
```diff
@@ -42,7 +52,8 @@
 	icon_state = "spickaxe"
 	item_state = "spickaxe"
 	worn_icon_state = "spickaxe"
-	toolspeed = 0.5 //mines faster than a normal pickaxe, bought from mining vendor
+	toolspeed = 0.5
+	stamina_use = 4 //costs less stamina because silver is lighter??
```
Comment (Rukofamicom) [inline code/modules/mining/equipment/mining_tools.dm]:
```suggestion
	stamina_use = 3 //costs less stamina because silver is lighter??
```

**Review by Rukofamicom — CHANGES_REQUESTED** [review]
That said, having tested it the stamina cost does feel a bit steep for what I was thinking. 

I do see my initial comment about the drill was incorrect now that I'm looking at the code as well.

**Comment by RDS88-beeman** [comment]
dreamseeker_KUCAQZZ0OA
this is borg proof, they still work

**Review by Rukofamicom — APPROVED** [review]

**Review by Tsar-Salat — APPROVED** [review]
I concur, looks fine. I do not think it will be a problematic alternative

**Pull request merged**